#! /bin/sh /usr/share/dpatch/dpatch-run
## 01gtkfont.dpatch by  <nr@eecs.harvard.edu>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Work around broken (and deprecated) GdkFont function

@DPATCH@
Index: uigtk2.ml
===================================================================
--- unison/uigtk2.ml	(revision 163)
+++ unison/uigtk2.ml	(working copy)
@@ -514,7 +514,7 @@
   ignore (lst#append ["Data received"]);
   ignore (lst#append ["File data written"]);
   let style = lst#misc#style#copy in
-  style#set_font (Lazy.force fontMonospaceMedium);
+  (* style#set_font (Lazy.force fontMonospaceMedium); *)
   for r = 0 to 2 do
     lst#set_row ~selectable:false r;
     for c = 1 to 3 do
